app/editor/src/features/admin/topic-score-rules/TopicScoreRuleList.tsx

```diff
@@ -104,195 +104,210 @@ export const TopicScoreRuleList: React.FC = () => {
             !request.isSilent && request.group.some((g) => g === 'content' || g === 'lookup')
           }
         >
-          {({ values, setFieldValue }) => (
-            <>
+          {({ values, setFieldValue, isSubmitting }) => (
```

Comment:
Attempted to get this page to vertically scroll properly but was unable... I hate css.